docs/02-concepts/02-data-contracts/index.md

```diff
@@ -209,30 +226,36 @@ using: `strm get data-contract strmprivacy/clickstream/1.0.0 -ojson | jq '.dataC
 </Tabs>
 
 1. `ref`: the reference to the data contract, comprised of a `handle` (globally unique, chosen by you / your
-  organization), `name`, and a [semantic](https://semver.org/) `version`.
+   organization), `name`, and a [semantic](https://semver.org/) `version`.
 2. `schemaRef`: the serialization schema reference, that follows the same `handle`, `name`, and `version`.
 3. `state`: whether the data contract is active and ready to be used
-  in [data pipelines](docs/02-concepts/01-data-processing/03-data-pipelines.md). More on states [here](#states).
+   in [data pipelines](docs/02-concepts/01-data-processing/03-data-pipelines.md). More on states [here](#states).
 4. `isPublic`: a data contract can be public, in which case all STRM
-  Privacy customers can use it, or it can belong to a certain
-  organization, and require organization credentials to use it.
+   Privacy customers can use it, or it can belong to a certain
+   organization, and require organization credentials to use it.
 5. `keyField`: the name of the field in the serialization schema that is used to
-  _tie_ events together. Typically, this is what determines a data
-  subject's (i.e. your users) session. If multiple events contain the same value for
-  the field that was specified as `keyField`, then
-  the [same encryption key and `keyLink`](docs/02-concepts/01-data-processing/01-pii-field-encryption.md) will be used to
-  encrypt the PII data.
-6. `piiFields`: the fields whose content in an event should be considered sensitive
-  (i.e. personally identifiable information), and should be encrypted.
-7. `validations`: the validations that should be performed on the content of specific
-  fields in an event.
-8. `dataSubjectField`: the name of the field in the serialization schema that is used to distinguish
-  data subjects from each other. Typically, this is a _customer id_ or a _user id_. This is used by the Data Subjects
-  API to keep [facilitate an interface](./04-data-subjects.md) that allows for easy retrieval of used `keyLink`s for a
-  specific data subject.
+   _tie_ events together. Typically, this is what determines a data
+   subject's (i.e. your users) session. If multiple events contain the same value for
+   the field that was specified as `keyField`, then
+   the [same encryption key and `keyLink`](docs/02-concepts/01-data-processing/01-pii-field-encryption.md) will be used
+   to
+   encrypt the PII data.
+6. `piiFields`: deprecated in favor of `fieldMetadata`
```

Comment:
Maybe you can create a new chip CSS class to mark this as deprecated and put some more attention to it (use `<div class="chip-optional"> <div class="chip-content">optional</div> </div>` inline to add a chip to this line).